### ISSUE TITLE
Bullet Physics shapes

### ISSUE THREAD

**Issue by Enhex** [godotengine/godot#17252]
Bullet provides collision shapes that Godot doesn't expose yet: cylinder and cone.

**Comment by akien-mga** [comment]
CC @AndreaCatania

**Comment by AndreaCatania** [comment]
Good call, in this moment we need to implement a feature for both 2D and 3D if we break this rule is easy for me to implement these things

**Comment by vnen** [comment]
I'm not sure what would be the equivalent of cylinder and cone shapes for 2D. If some feature is only relevant for 3D, I'd say to go for it, especially if it's easy.

**Comment by aaronfranke** [comment]
Bullet uses vertical-by-default collision meshes. We need to fix https://github.com/godotengine/godot/issues/5813 first.

**Comment by akien-mga** [comment]
Cylinder shape implemented by #19540.

**Comment by aaronfranke** [comment]
Does anyone want cone shapes? I personally have no use for them but some may find it useful.

**Comment by akien-mga** [comment]
Let's wait for anyone to request them, we don't implement features unless users need them.
